deployment/Dockerfile.prod

```diff
@@ -0,0 +1,4 @@
+FROM segment/chamber:2 AS chamber
```

Comment:
I don't like this as it:

a) changes base image. Is it alpine? Is it debian? 
b) we don't know the exact version installed.